src/site/_includes/partials/analytics.njk

```diff
@@ -10,6 +10,11 @@ ga('set', 'transport', 'beacon');
 ga('set', 'anonymizeIp', true);
 ga('set', 'page', window.location.pathname);
 ga('set', '{{ site.analytics.dimensions.TRACKING_VERSION }}', '{{ site.analytics.version}}');
+try {
+  ga('set', '{{ site.analytics.dimensions.NAVIGATION_TYPE }}', performance.getEntriesByType('navigation')[0].type);
```

Comment:
Most modern browsers support `performance.getEntriesByType()`, but to avoid issues with older browsers, it's safer to just wrap this in a try/catch to prevent errors breaking the rest of the analytics initialization code.